poetry/core/json/schemas/poetry-schema.json

```diff
@@ -55,8 +55,19 @@
       "$ref": "#/definitions/maintainers"
     },
     "readme": {
-      "type": "string",
-      "description": "The path to the README file"
+      "anyOf": [
+        {
+          "type": "string",
+          "description": "The path to the README file"
```

Comment:
Descriptions here should be sentences. For some reason the existing description was not -- add some periods and we should be good.